test/e2e/common_test.go

```diff
@@ -249,6 +249,11 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 	}
 	os.Setenv("DISABLE_HC_SYSTEMD", "true")
 
+	dbBackend := "boltdb"
+	if os.Getenv("PODMAN_DB") == "sqlite" {
```

Comment:
Could you add a note on the new env var to `tests/README.md` ?

Reply:
There's absolutely no documentation of test environment variables in there, so I'm not sure it's a good place. Can't find any documentation of testing environment in test/ actually.

Reply:
The tests are used outside of upstream CI, so I think it's important to leave breadcrumbs.  If there are more undocumented variables, we should document them as well (in another PR for sure).

Reply:
There are something like 2 dozen undocumented environment variables, unfortunately

Reply:
just put the description right here imho